<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionDialog</class>
 <widget class="QDialog" name="SectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>531</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理部门</string>
  </property>
  <widget class="QGroupBox" name="box_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>管理部门详细信息</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
       <property name="spacing">
        <number>35</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_code">
           <property name="text">
            <string>部门编号：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_code">
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>部门名称：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_name">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
       <property name="spacing">
        <number>35</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_leader">
           <property name="text">
            <string>部门领导：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_leader">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_location">
           <property name="text">
            <string>办公地点：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_location">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_setting">
         <property name="text">
          <string>组织机构：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_setting">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_rule">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>规章制度：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="edit_rule">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_select">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>查询</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
     <property name="spacing">
      <number>35</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_condition">
         <property name="text">
          <string>查询条件：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_condition">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_content">
         <property name="text">
          <string>查询内容：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_content">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_select">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>56</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>56</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="list_section">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>SectionDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
